packages/application-extension/src/index.tsx

```diff
@@ -45,10 +45,14 @@ import React from 'react';
 
 class WebRtcProvider extends WebrtcProvider implements IDocumentProvider {
   constructor(options: IDocumentProviderFactory.IOptions & { room: string }) {
-    super(`${options.room}${options.guid}`, options.ymodel.ydoc);
+    super(`${options.room}${options.path}`, options.ymodel.ydoc);
     this.awareness = options.ymodel.awareness;
   }
 
+  setPath() {
+    // TODO: this seems super useful
```

Comment:
This was added in https://github.com/jupyterlab/jupyterlab/pull/10470 to support renaming of files.

It would be fine to tackle this separately as it might require a different logic than upstream (we don't have that common websocket endpoint)

Reply:
hm... is a no-op fine, or do we need some kind of message?

Reply:
> For the usernames and colors, we'll need to implement something similar to what is upstream since we replace the provider: jupyterlab/jupyterlab#10411

Sounds fun. Alas, even though Callisto is big enough to have moons, we haven't found any. Maybe names of asteroids? Or entirely made-up moons from various sci-fi universes?

So do we collect all of the RTC issues into one follow-on?

Reply:
> So do we collect all of the RTC issues into one follow-on?

Yes better to fix them outside of this PR. The username / color should be straightforward normally.